docs/data/product/australian-national-spectral-database/_data.yaml

```diff
@@ -3,20 +3,23 @@
 
 # Overview
 
-title: Australian National Spectral Database
-long_title: Australian National Spectral Database
+short_name: Australian National Spectral Database
+full_technical_name: null
 header_image: /_files/cmi/spectral_2.jpg
 version: 3.3.0
 is_latest_version: true
 latest_version_link: null
 is_provisional: false
-product_type: Data service
-spatial_data_type: null
+type:
+  product_type: Data service
+  spatial_data_type: null
```

Comment:
@robbibt Would you know if this is a Raster or Vector product (or neither)?

Reply:
Kind of more a service - really a repository or data base into which users can deposit or access spectral information.